packages/integrations/src/interfaces/dns-hole-summary/dns-hole-summary-types.ts

```diff
@@ -0,0 +1,6 @@
+export interface DnsHoleSummary {
+  domainsBeingBlocked: number;
+  adsBlockedToday: number;
+  adsBlockedTodayPercentage: number;
+  dnsQueriesToday: number;
```

Comment:
Couldn't percentage be computed by simply doing AdsToday / QueriesToday * 100 ?

Reply:
Yes but it's already provided by the api anyway